node/node.go

```diff
@@ -546,7 +546,7 @@ func (node *Node) CalculateInitShardState() (err error) {
 
 	// Get genesis epoch shard state from chain
 	blockNum := node.Blockchain().CurrentBlock().NumberU64()
-	node.Consensus.SetMode(consensus.Listening)
+	node.Consensus.SetMode(values.PBFTListening)
```

Comment:
Once again, `values.PBFTListening` is a duh-boilerplate-y name.  `consensus.PBFTListening` or `consensus.Listening` flows more natural.  (Or even better, if PBFT algorithm were factored into its own package, it could have been `pbft.Listening`… boom.)

Reply:
agreed